Gulpfile.ts

```diff
@@ -748,7 +745,7 @@ gulp.task("runtests",
 const nodeServerOutFile = "tests/webTestServer.js";
 const nodeServerInFile = "tests/webTestServer.ts";
 gulp.task(nodeServerOutFile, false, [servicesFile], () => {
-    const settings: tsc.Settings = getCompilerSettings({}, /*useBuiltCompiler*/ true);
+    const settings: tsc.Settings = getCompilerSettings({module: "commonjs"}, /*useBuiltCompiler*/ true);

```

Comment:
Yeah - this style is inconsistently enforced, I have instances both with and without spaces in the compiler. We need a lint rule for this, one way or the other, if people are going to display a preference.